app/src/main/java/com/geeksville/mesh/ui/NodeInfo.kt

```diff
@@ -149,20 +149,44 @@ fun NodeInfo(
                             bias = 0F,
                             startMargin = 8.dp,
                             endMargin = 8.dp,
-
-                        )
+                            )
                         width = Dimension.preferredWrapContent
                     },
                     text = nodeName,
                     style = style,
                     textDecoration = TextDecoration.LineThrough.takeIf { isIgnored },
                 )
 
+                val hwInfoString = thisNodeInfo?.user?.hwModelString
```

Comment:
just noticed this should be using `thatNodeInfo`, otherwise it will show our local node's hardware for everyone.

let me know if you want to follow up with a fix.

Reply:
@andrekir https://github.com/meshtastic/Meshtastic-Android/pull/1123 fix here 🙃